packages/eslint-plugin-template/src/rules/i18n.ts

```diff
@@ -522,26 +522,26 @@ function isBoundTextAllowed(
   return !PL_PATTERN.test(text) || allowedBoundTextPattern.test(text);
 }
 
-function isTemplate(ast: unknown): ast is TmplAstTemplate {
-  return ast instanceof TmplAstTemplate;
+function isNgTemplate(ast: unknown): ast is TmplAstTemplate {
+  return ast instanceof TmplAstTemplate && ast.tagName === 'ng-template';
```

Comment:
This is the actual fix, the rest is just a refactor to rename the method to something better.